codegenerator/cli/templates/static/codegen/src/db/DbFunctions.res

```diff
@@ -98,10 +98,10 @@ module ChainMetadata = {
     @as("chain_id") chainId: int,
     @as("block_height") blockHeight: int,
     @as("start_block") startBlock: int,
-    @as("end_block") endBlock: option<int>,
-    @as("first_event_block_number") firstEventBlockNumber: option<int>,
-    @as("latest_processed_block") latestProcessedBlock: option<int>,
-    @as("num_events_processed") numEventsProcessed: option<int>,
+    @as("end_block") endBlock: Js.Nullable.t<int>,
```

Comment:
Shouldn't it be Js.Null.t<int> instead?

Reply:
Happy to use Js.Null.t though if I'm being silly

Reply:
I think it's fine. But I'd like to ask to leave a comment that it's probably Null and we use Nullable only because we didn't confirm it yet. So in the future we don't have hard assumptions that there's an undefined incoming

Reply:
@DZakh done 🙏🏼